weaviate/values-override.yml

```diff
@@ -0,0 +1,51 @@
+ingress:
+  enabled: true
+  paths:
+    - /*
+  hosts:
+    - chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+image:
+  tag: {{ .Values.envs.imageTag }}
+
+appLabels:
+  region: ""
+
+replicas: {{ .Values.envs.replicaCount }}
+
+storage:
+  size: {{ .Values.envs.storageSize }}
+
+modules:
+  text2vec-openai:
+    enabled: true
+    apiKey: '{{ .Values.envs.MODULES_API_KEY }}'
+  generative-openai:
+    enabled: true
+    apiKey: '{{ .Values.envs.MODULES_API_KEY }}'
+
+authentication:
+  anonymous_access:
+    enabled: false
+  apikey:
+    enabled: true
+    allowed_keys:
+      - {{ .Values.envs.AUTHENTICATION_API_KEY }}
+    users:
+      - admin
+
+query_defaults:
+  limit: {{ .Values.envs.QUERY_LIMIT }}
+
+service:
+  name: weaviate
```

Comment:
this should be named ms-labs-weaviate
and should be accordingly updated in other places to match lables